src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -36,25 +36,364 @@
+        List<AfflMappingWrapperByNameAndLabel> wrapperList =
+            new List<AfflMappingWrapperByNameAndLabel>();
+
+        List<String> contactPrimaryAffiliationFieldNameList = 
+            AFFL_MultiRecordType_TEST.getContactPrimaryAffiliationFieldLabels();
+
+        wrapperList.add(
+            new AfflMappingWrapperByNameAndLabel(
+                UTIL_Describe_API.getHhAccRecTypeID(),
+                contactPrimaryAffiliationFieldNameList[0]
+            )
+        );
+
+        wrapperList.add(
+            new AfflMappingWrapperByNameAndLabel(
+                UTIL_Describe_API.getBizAccRecTypeID(),
+                contactPrimaryAffiliationFieldNameList[1]
+            )
+        );
+
+        return wrapperList;
+    }
+
+    /**************************************************************************
+    * @description Initializes the mappings based on the affiliation mapping
+    * wrappers passed to it
+    * @param affiliationMappingWrapperList The list of account record type ids to get
+    * mappings for.
+    * @return The List of Affiliation Mappings to insert.
+    **************************************************************************/
+    private static List<AffiliationMappingWrapper> populateRecordTypeNamesForWrappers(
+        List<AffiliationMappingWrapper> affiliationMappingWrapperList
+    ) {
+        System.assertEquals(false, affiliationMappingWrapperList.size() < 2,
+            '2 Account Record Types were not found for affiliation mappings');
+            
+        Map<Id, Schema.RecordTypeInfo> accountRecordTypeInfosByIds =
+            AFFL_MultiRecordType_TEST.getIdToAccountRecordTypesMap();
+
+        for(AffiliationMappingWrapper affiliationMappingWrapper : affiliationMappingWrapperList) {
+            affiliationMappingWrapper.setRecordTypeNamesFromId(accountRecordTypeInfosByIds);
+        }
+
+        return affiliationMappingWrapperList;
+    }
+
+    /**************************************************************************
+    * @description Get record type infos for an account
+    * @return The List of Affiliation Mappings to insert.
+    **************************************************************************/
+    private static Map<Id, Schema.RecordTypeInfo> getIdToAccountRecordTypesMap() {
+        Schema.DescribeSObjectResult accountDescribe =
+            UTIL_Describe.getObjectDescribe('Account');
+
+        return accountDescribe.getRecordTypeInfosByID();
+    }
+
+    /**************************************************************************
+    * @description Creates affiliation mappings from wrappers, setting the
+    * related mapping to the wrapper for later reference
+    * @param affiliationMappingWrapperList The list of mapping wrappers to
+    * create mappings for
+    * @return The affiliation mappings created 
+    **************************************************************************/
+    private static List<Affl_Mappings__c> createAffiliationMappings(
```

Comment:
As a separate note, we can potentially put these into UTIL_UnitTestData_TEST. Well actually, it won't hurt to put them now since that class is only used for unit tests but sure let's keep them here for now.

Reply:
It wouldn't hurt. I'd like us to look at this comprehensively as this gets pretty zany and centralizing this by the feature might be advantageous for us rather than adding to a list of unit test data in general.